README.md

```diff
@@ -65,8 +65,9 @@ http {
                 -- Currently, we can not have the upstream real network address
                 ------------------------------------------------------
                 skywalking_tracer:start("upstream service")
```

Comment:
I think we should consider recommending `skywalking_tracer:start()`(no parameter required) here as we separate `inject` method?

Reply:
If `skywalking_tracer:inject(ngx.ctx.exitSpan)` always works, then it seems pointless to keep manual `upstream service`.